src/components/atoms/ActionButtonV2/ActionButtonV2.css

```diff
@@ -0,0 +1,44 @@
+div {
+  font-family: var(--font-family);
+}
+
+.action-button-container,
+.action-button-container > a,
+.action-button-container > a > .abutton {
+  width: 100%;
+  height: 100%;
+}
+
+.top-icon {
+  padding-top: 1em;
+}
+
+.abutton-title {
+  margin-bottom: var(--abutton-title-spacer-y);
+  font-weight: 900;
```

Comment:
We might as well remove this because it has no effect on the slotted element. We can let the light DOM style the element.